ts/store/reducers/entities/messages/messagesById.ts

```diff
@@ -21,11 +21,11 @@ export type MessageDetailsByIdState = Readonly<{
   id: string;
   createdAt: string;
   markdown: string | undefined;
-  paymentData: PaymentData;
+  paymentData?: PaymentData;
   serviceName: string;
   serviceOrganizationName: string;
   serviceDepartmentName: string;
-  subject: string;
+  subject?: string;
```

Comment:
`subject` is required, it shouldn't be optional, check the `MessageWithContent` type

Reply:
Actually, `markdown`, `payment_data` and `subject` are typed as optional.
```javascript
const MessageWithContentR = t.interface({
  created_at: Timestamp,
  id: t.string,
  sender_service_id: t.string
});

const MessageWithContentO = t.partial({
  markdown: MessageBodyMarkdown,
  payment_data: PaymentData,
  subject: MessageSubject
});

export const MessageWithContent = t.exact(
  t.intersection(
    [MessageWithContentR, MessageWithContentO],
    "MessageWithContent"
  )
);
```
Or am I get it wrong?